react/features/invite/actions.js

```diff
@@ -85,7 +89,8 @@ export function invite(invitees: Array<Object>) {
             // filter all rooms and users from {@link invitesLeftToSend}.
             const peopleInvitePromise
                 = invitePeopleAndChatRooms(
-                    inviteServiceUrl,
+                    callFlowsEnabled
+                        ? inviteServiceCallFlowsUrl : inviteServiceUrl,
```

Comment:
It's necessary to support both urls? Preferring inviteServiceCallFlowsUrl over inviteServiceUrl isn't an option? Not using a config to determine which config to use would be nice if possible.